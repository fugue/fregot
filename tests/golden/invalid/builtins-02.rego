# '-' operator type mismatches {set, number} <-> {number, set}
package fregot.tests.invalid.builtins_02

test_number_minus_set {
  0 - set()
}

test_set_minus_number {
  set() - 0
}
