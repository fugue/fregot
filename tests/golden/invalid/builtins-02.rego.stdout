passed: 0, failed: 0, errored: 2
fregot.tests.invalid.builtins_02.test_number_minus_set:
fregot ([34meval error[0m):
  "builtins-02.rego" (line 5, column 3):
  builtin error:

    5|   0 - set()
         [31m^^^^^^^^^[0m

  Expected number but got set

  Stack trace:
    rule [33mfregot.tests.invalid.builtins_02[33m.[0mtest_number_minus_set[0m at tests:1:1
fregot.tests.invalid.builtins_02.test_set_minus_number:
fregot ([34meval error[0m):
  "builtins-02.rego" (line 9, column 3):
  builtin error:

    9|   set() - 0
         [31m^^^^^^^^^[0m

  Expected set but got number

  Stack trace:
    rule [33mfregot.tests.invalid.builtins_02[33m.[0mtest_set_minus_number[0m at tests:1:1
